iset.mm

```diff
@@ -41635,6 +41635,22 @@ arbitrary sets or real numbers will not be provable (without the law of
       ( cvv wcel cres resexg ax-mp ) ADEABFDECABDGH $.
   $}
 
+  $( Class restriction distributes over intersection.  (Contributed by FL,
+     6-Oct-2008.) $)
+  resindm $p |- ( Rel A -> ( A |` ( B i^i dom A ) ) = ( A |` B ) ) $=
+    ( wrel cres cdm cin resdm resindi incom inres inidm reseq1i 3eqtrri 3eqtr4g
```

Comment:
The comment does not correspond to the statement.

A precise description would be verbose, so maybe the less precise: "When restricting a relation, intersecting [the restrictor] with the domain of the relation has no effect."

Reply:
Yeah I agree that "distributes over" is pretty far from what is happening here.

I think "When restricting a relation, intersecting with the domain of the relation has no effect." is probably about the right balance of conveying the idea without getting too bogged down. These comments are (generally speaking) designed to be read in conjunction with the formal statement of the theorem rather than being something you'd read instead of the theorem.

I'll make that change in both set.mm and iset.mm.